packages/web/src/app/doc-pages/about/tutorial/blazor-tutorial/blazor-tutorial.component.html

```diff
@@ -0,0 +1,147 @@
+<!--BASIC TUTORIAL-->
+<div class="e-alert e-alert--info e-mt-40" role="alert">
```

Comment:
```suggestion
<div class="e-alert e-alert--info e-mt-40" role="note">
```
Or just remove the role 🤷🏽 its not that important in this context